### ISSUE TITLE
Add torch backend for sits [R]

### ISSUE THREAD

**Issue by TomAugspurger** [microsoft/planetary-computer-containers#36]
https://github.com/conda-forge/staged-recipes/pull/13992 was an attempt to package with conda-forge. Ran into an issue described in https://github.com/mlverse/torch/issues/341 (perhaps something about how mlverse/torch is bundling binaries?). Doesn't look straightforward to do via conda / conda-forge.

**Comment by gilbertocamara** [comment]
I don't think you need to write a conda package. The normal CRAN install works.

**Comment by gilbertocamara** [comment]
Dear @TomAugspurger, I have interacted with Daniel Falbel, who is the `torch` maintainer for R. My guess is that the problems arise from the fact that CRAN does not allow packages to include external binary libraries. The package `torch` for R needs two C++ libraries (libtorch and liblantern) that have to be loaded on runtime. Thus, it is not self contained.  The first time any `torch` function is called on an R script, these libraries are called.

**Comment by gilbertocamara** [comment]
If you want, we could prepare a Docker container with typical packages required by R applications that use EO data, e.g., `rgdal`, `rstac`, `sf`, etc.

**Comment by TomAugspurger** [comment]
Thanks. I pursued the current conda-based setup since we need to have the UI packages (like jupyterlab / notebook) which are a mixture of javascript and Python, in addition to the analysis libraries, and I didn't want to maintain multiple build systems for the various environments.

But if we're trying to push more complex packages into the R environment, it might make sense to switch how we do that environment. I think we could use or build off something like @cboettig's rocker project.

**Comment by gilbertocamara** [comment]
Hi @TomAugspurger, we will work with our team to prepare an R container similar to rocker.
